solidity/contracts/BeaconBackportRewards.sol

```diff
@@ -18,110 +18,95 @@ import "./Rewards.sol";
 import "./KeepRandomBeaconOperator.sol";
 import "./TokenStaking.sol";
 
+/// @title KEEP Random Beacon Signer Subsidy Rewards for the May release.
+/// @notice Contract distributing KEEP rewards to Random Beacon signers from
+/// May KeepRandomBeaconOperator contract:
+/// https://etherscan.io/address/0x70F2202D85a4F0Cad36e978976f84E982920A624
+///
+/// We use a separate contract for those rewards as the previous version of
+/// KeepRandomBeaconOperator did not have all the functions BeaconRewards uses.
+///
+/// Groups from May release of KeepRandomBeaconOperator contract can claim they
+/// rewards at any time.
 contract BeaconBackportRewards is Rewards {
-    uint256[] lastGroupOfInterval;
-    mapping(uint256 => bool) excludedGroups;
-    uint256 excludedGroupCount;
+
+    // Beacon genesis date, 2020-05-11, is the interval start.
+    // https://etherscan.io/tx/0x5c0387a2402be57dae95d5f5c3745afb3a770462df13fceccf3967a1eecf6136
+    uint256 internal constant beaconIntervalStart = 1589155200;
+
+    // We are going to have one interval, with a weight of 100%.
+    uint256[] internal beaconIntervalWeight = [100];
+    uint256 internal constant lastInterval = 0;
+
+    // 135 days between the genesis of the old and the new random beacon
+    // contract versions:
+    // https://etherscan.io/tx/0x5c0387a2402be57dae95d5f5c3745afb3a770462df13fceccf3967a1eecf6136
+    // https://etherscan.io/tx/0xe2e8ab5631473a3d7d8122ce4853c38f5cc7d3dcbfab3607f6b27a7ef3b86da2
+    uint256 internal constant beaconTermLength = 135 days;
```

Comment:
1600905600 - 1589155200 = 11750400sec which is 136 days

Reply:
Good catch, should be 136.